src/main/java/de/caritas/cob/consultingtypeservice/api/converter/TopicGroupConverter.java

```diff
@@ -0,0 +1,31 @@
+package de.caritas.cob.consultingtypeservice.api.converter;
+
+import de.caritas.cob.consultingtypeservice.api.model.TopicGroupEntity;
+import de.caritas.cob.consultingtypeservice.api.model.TopicGroupsDTO;
+import de.caritas.cob.consultingtypeservice.api.model.TopicGroupsDTOData;
+import de.caritas.cob.consultingtypeservice.api.model.TopicGroupsDTODataItemsInner;
+import java.util.Collection;
+import java.util.stream.Collectors;
+import lombok.val;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TopicGroupConverter {
+
+  public TopicGroupsDTO toTopicGroupsDTO(Collection<TopicGroupEntity> topicGroups) {
+    val tg = new TopicGroupsDTO();
```

Comment:
could you pls change tg to toggleGroups ? according to clean code, such abbreviations need to ba avoided

Reply:
I sure can, but I don't consider it to be good practice. Why?

* I believe that variable name length should be proportional to the scope of the variable. As this scope is small (our eyeballs can find the variable definition extremely fast), the var name length should be small.
* It introduces additional noise to already noisy language, which distracts me when figuring out what the business logic does

Reply:
Better readibility is not a noise, so I cannot agree to that :) 

I still think it's better to use non abbreviated var names. Looking at the code, and seeing te1 or tg1, I don't know what this variable holds unless I actually see the definition. So the code is less readable. 
Also please have a look at this section, they explain why it's better to avoid abbreviations:
https://readlearncode.com/code-and-stuff/clean-code-variables/

For tests and in the scope you have added it it's ok, but I prefer to avoid it.
One additional arugment: if you extract variable in intellij, it proposes meaningful names instead of abbreviations.

You don't need to change it now, but I'd prefer if you follow this convention for subsequent code changes.